dataland-frontend/src/components/wrapper/AuthorizationWrapper.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <div v-if="hasUserRequiredRole">
```

Comment:
Is it intended that there is no edit button for me for this dataset, even though I am certified data owner?:

![image](https://github.com/d-fine/Dataland/assets/99255296/3daafa1d-13a7-40c4-b0e4-b1f70d3de538)

Reply:
Not intended, checking this

Reply:
Wenn ich das dataset anschaue und dann auf "Edit" klicke, dann sehe ich kurz "you do not have permission".
Nach einem Augenblick verschwindet das jedoch und ich sehe die upload page.

Reply:
Aber es sollte ein Loading Spinner zu sehen sein

Reply:
Das Problem ist immer noch da (lokal getestet)

Reply:
Ich schließe die conversation hier, weil das Thema bereits in einer anderen conversation diskutiert wird